modules/substrate/src/lib.rs

```diff
@@ -202,17 +221,48 @@ decl_module! {
 		/// This function is only allowed to be called from a trusted origin and writes to storage
 		/// with practically no checks in terms of the validity of the data. It is important that
 		/// you ensure that valid data is being passed in.
+		///
+		/// There's also nothing stopping Owner or Root from calling this function twice (which
+		/// would be bad). Ensure this isn't the case before submitting the call.
```

Comment:
I'm strongly in favor of adding a check here and then adding `reinitialize` method (please create a separate issue for this).

Re-initialization should be heavily tested to make sure the bridge pallet state maintains consistent if such event occurs.